ci-operator/config/Azure/ARO-HCP/Azure-ARO-HCP-main__periodic.yaml

```diff
@@ -12,176 +12,55 @@ images:
-    - chain: aro-hcp-provision-create-functional-cluster
 - as: delete-expired-integration-resource-groups
   cron: 7,37 * * * *
   steps:
-    env:
-      CUSTOMER_SUBSCRIPTION: ARO SRE Team - INT (EA Subscription 3)
-    test:
-    - ref: aro-hcp-provision-delete-all-expired-resource-groups
-- as: integration-e2e-parallel
-  cron: 0 */4 * * *
-  steps:
-    allow_best_effort_post_steps: true
-    env:
-      ARO_HCP_SUITE_NAME: integration/parallel
-      CUSTOMER_SUBSCRIPTION: ARO SRE Team - INT (EA Subscription 3)
-      LOCATION: uksouth
-    post:
-    - ref: aro-hcp-provision-aro-hcp-gather-extra
-    - chain: aro-hcp-provision-teardown-cluster
+    cluster_profile: aro-hcp-int
     test:
-    - ref: aro-hcp-tests-run-aro-hcp-tests
-- as: create-aro-hcp-in-stage
-  cron: 0 */4 * * *
-  steps:
-    allow_best_effort_post_steps: true
-    env:
-      CLUSTER_NAME: periodic-stage-cluster
-      CUSTOMER_RESOURCE_GROUP_PREFIX: periodic-stage-customer-rg
-      LOCATION: uksouth
-      SUBSCRIPTION: ARO HCP - STAGE testing (EA Subscription)
-    post:
-    - ref: aro-hcp-provision-aro-hcp-gather-extra
-    - chain: aro-hcp-provision-teardown-cluster
-    pre:
-    - ref: aro-hcp-provision-generate-resource-group-name
-    test:
-    - chain: aro-hcp-provision-create-functional-cluster
-- as: stage-e2e-parallel
-  cron: 0 */4 * * *
-  steps:
-    allow_best_effort_post_steps: true
-    env:
-      ARO_HCP_SUITE_NAME: stage/parallel
-      CUSTOMER_SUBSCRIPTION: ARO HCP - STAGE testing (EA Subscription)
-      LOCATION: uksouth
-    post:
-    - ref: aro-hcp-provision-aro-hcp-gather-extra
-    - chain: aro-hcp-provision-teardown-cluster
-    test:
-    - ref: aro-hcp-tests-run-aro-hcp-tests
+    - ref: aro-hcp-deprovision-expired-resource-groups
 - as: delete-expired-stage-resource-groups
   cron: 7,37 * * * *
   steps:
-    env:
-      CUSTOMER_SUBSCRIPTION: ARO HCP - STAGE testing (EA Subscription)
-    test:
-    - ref: aro-hcp-provision-delete-all-expired-resource-groups
-- as: create-aro-hcp-in-prod-uksouth
-  cron: 0 */4 * * *
-  steps:
-    allow_best_effort_post_steps: true
-    env:
-      CLUSTER_NAME: periodic-prod-uksouth-cluster
-      CUSTOMER_RESOURCE_GROUP_PREFIX: periodic-prod-uksouth-customer-rg
-      LOCATION: uksouth
-      SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
-    post:
-    - ref: aro-hcp-provision-aro-hcp-gather-extra
-    - chain: aro-hcp-provision-teardown-cluster
-    pre:
-    - ref: aro-hcp-provision-generate-resource-group-name
-    test:
-    - chain: aro-hcp-provision-create-functional-cluster
-- as: prod-uksouth-e2e-parallel
-  cron: 0 */4 * * *
-  steps:
-    allow_best_effort_post_steps: true
-    env:
-      ARO_HCP_SUITE_NAME: prod/parallel
-      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
-      LOCATION: uksouth
-    post:
-    - ref: aro-hcp-provision-aro-hcp-gather-extra
-    - chain: aro-hcp-provision-teardown-cluster
+    cluster_profile: aro-hcp-stg
```

Comment:
I checked the vault, so you have created the vault for the staging and prod? So later if I want to move staging and prod e2e test to the new tenant, I just need to update the vault?